src/lib/optimizer/strategy/stored_table_column_alignment_rule.cpp

```diff
@@ -10,10 +10,14 @@
 #include "logical_query_plan/lqp_utils.hpp"
 
 namespace {
+using namespace opossum;  // NOLINT
```

Comment:
The changes to this file are not tested yet, are they?

Reply:
No tests for the STN-alignment across subqueries yet. I'll try to add some.